src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/IncomingMessageConsumer.java

```diff
@@ -45,11 +45,10 @@ void close() {
     public void readAndProcess() {
         for (Message msg : consumer.consume()) {
             try {
-                logger.info(msg.toString());
+                logger.info(String.format("Message offset %s retrieved, processing", msg.getOffset()));
```

Comment:
would message id be a useful thing to log out as well?

Reply:
we don't actually have access to the message id, there is a message.getKey() field, but that returns null.